src/Compute/Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetRestartMethod.cs

```diff
@@ -46,7 +46,15 @@ public override void ExecuteCmdlet()
                     string vmScaleSetName = this.VMScaleSetName;
                     System.Collections.Generic.IList<string> instanceIds = this.InstanceId;
 
-                    var result = VirtualMachineScaleSetsClient.RestartWithHttpMessagesAsync(resourceGroupName, vmScaleSetName, instanceIds).GetAwaiter().GetResult();
+                    Rest.Azure.AzureOperationResponse result;
+                    if (NoWait.IsPresent)
+                    {
+                        result = VirtualMachineScaleSetsClient.BeginRestartWithHttpMessagesAsync(resourceGroupName, vmScaleSetName, instanceIds).GetAwaiter().GetResult();
+                    }
+                    else
+                    {
+                        result = VirtualMachineScaleSetsClient.RestartWithHttpMessagesAsync(resourceGroupName, vmScaleSetName, instanceIds).GetAwaiter().GetResult();
+                    }
```

Comment:
see above, and apply this everywhere this pattenr occurs - if we can verify it once, then this is OK, otherwise we should hold off.

Reply:
Done